src/NetEscapades.EnumGenerators/DiagnosticHelper.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.CodeAnalysis;
+
+namespace NetEscapades.EnumGenerators;
+
+public static class DiagnosticHelper
+{
+    public static readonly DiagnosticDescriptor EnumInGenericType = new(
+#pragma warning disable RS2008 // Enable Analyzer Release Tracking
+        id: "NEEG002",
```

Comment:
We should start this at `NEEG001` seeing as it's the first diagnostic being added